.circleci/config.yml

```diff
@@ -269,24 +269,30 @@ jobs:
           ./sst_dump --help | grep -E -q 'Supported compression types: kNoCompression$' # Verify no compiled in compression
       - post-steps
 
-  build-linux-shared_lib-alt_namespace-status_checked:
+  build-linux-static_lib-alt_namespace-status_checked:
```

Comment:
Are these changes from shared to static strictly needed or just to ensure test coverage for different LIB_MODE?

Reply:
Covering different LIB_MODEs in CI

Reply:
Actually, the other two cases (build-linux-gcc-7-with-folly, build-linux-gcc-11-no_test_run) I changed because of obscure failures. Probably fixable by installing the right libraries or something, but didn't want to dig into it. I can add comments for that.

Reply:
Specifically, linker failures:
https://app.circleci.com/pipelines/github/facebook/rocksdb/23961/workflows/52b4eb91-6d58-4566-800f-0ccb0d18b574/jobs/562894
https://app.circleci.com/pipelines/github/facebook/rocksdb/23961/workflows/0612336f-216c-41da-8d10-02d545df9749/jobs/562868